lib/matplotlib/mpl-data/stylelib/cheatsheet_gallery.mplstyle

```diff
@@ -0,0 +1,19 @@
+# from the Matplotlib cheatsheet as used in our gallery:
```

Comment:
We could?  Whichever is easiest - I'm not sure about user examples that don't have a public stylesheet....

Reply:
IMHO it's ok to make the style public. If somebody wants to use that style, that's ok.

Is `cheatsheet_gallery` a good public name? I'm not sure if `cheatsheet` is a helpful term here, in particular also because the style looks a bit like the cheatsheets but is not used there. What about `mpl_plot_gallery`?

Reply:
Done

Reply:
I guess it's okay to leave public, but I'm not sure we want to say that it will follow any API guarantees.

Reply:
Well, thats a good point.  Happy to do a follow up to fix that...